docs/app/src/docs.js

```diff
@@ -53,8 +51,8 @@ angular.module('DocsController', [])
    Initialize
    ***********************************/
 
-  $scope.versionNumber = angular.version.full;
-  $scope.version = angular.version.full + '  ' + angular.version.codeName;
+  $scope.versionNumber = NG_VERSION.full;
+  $scope.version = NG_VERSION.full + ' ' + NG_VERSION.codeName;
```

Comment:
In the master build, I don't think that there is a `codeName` etc...
Don't we need to do some logic similar to what was removed in line 11 above?